apps/desktop-wallet/src/hooks/useWalletLock.ts

```diff
@@ -72,9 +76,8 @@ const useWalletLock = () => {
 
     try {
       await migrateUserData(encryptedWallet.id, password, version)
-    } catch (e) {
-      console.error(e)
-      posthog.capture('Error', { message: 'User data migration failed ' })
+    } catch (error) {
+      sendAnalytics({ type: 'error', error, message: 'User data migration failed', isSensitive: true })
```

Comment:
This should be fine. I wondered if the password could be leaked somehow, but I don't think so.

Reply:
I think it's not fine, checking the implementation of the function there are many exceptions that could potentially be thrown.